src/Symfony/Component/Dotenv/README.md

```diff
@@ -4,10 +4,32 @@ Dotenv Component
 Symfony Dotenv parses `.env` files to make environment variables stored in them
 accessible via `$_SERVER` or `$_ENV`.
 
+Getting Started
+---------------
+
+```
+$ composer require symfony/dotenv
+```
+
+```php
+use Symfony\Component\Dotenv\Dotenv;
+
+$dotenv = new Dotenv();
+$dotenv->load(__DIR__.'/.env');
+
+// you can also load several files
+$dotenv->load(__DIR__.'/.env', __DIR__.'/.env.dev');
+
+// overwrites existing env variables
+$dotenv->overload(__DIR__.'/.env');
+
+// loads .env, .env.local, and .env.$APP_ENV.local or .env.$APP_ENV
+$dotenv->loadEnv(__DIR__.'/.env');
+```
+
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/components/dotenv.html)
```

Comment:
for my understanding, are we really going to delete this doc page?

Reply:
Yes, I think so. The API of this component is quite simple (only 3 methods of the same class where documented - which are now shown in the example). In symfony/symfony-docs#13706, I've moved the `.env` syntax to the main configuration guide. Finally, the `loadEnv` logic is explained in both the main guide and the PHPdocs.

So I think this is a component where no knowledge is lost when removing its component docs.